simd/src/Kokkos_SIMD.hpp

```diff
@@ -84,33 +95,45 @@ namespace simd_abi {
 namespace Impl {
 
 #if defined(KOKKOS_ARCH_AVX512XEON)
+template <class T>
 using host_fixed_native = avx512_fixed_size<8>;
```

Comment:
I don't think I agree with this right now. We may need to rethink this alltogether but this is very specific choices based on the number of `double` values fitting into a simd register.

Reply:
Please see comment https://github.com/kokkos/kokkos/pull/7807#discussion_r1979528085